backend/src/test/java/gov/cdc/usds/simplereport/service/TestOrderServiceTest.java

```diff
@@ -2504,27 +2549,30 @@ private List<TestEvent> makedata() {
     return testEvents;
   }
 
-  private List<TestEvent> makeSpecificDiseaseData(SupportedDisease disease) {
+  private List<TestEvent> makeSpecificDiseaseData(SupportedDisease disease, Facility site) {
```

Comment:
Added a parameter to pass in a specific site so that one of the tests can use the global instance variable facility site rather than a new disease specific facility created below